src/main/webapp/js/map.js

```diff
@@ -0,0 +1,35 @@
+      let map;
+      function createMap(){
+          map = new google.maps.Map(document.getElementById('map'), {
+            center: {lat: 37.422, lng: -122.084},
+            zoom: 16
+          });
+
+          // Add a marker at Googleplex
```

Comment:
Add a marker at Googleplex (line 9) and Add a marker for the dog park (line 21) are quite similar. You may consider refractor it into a function to avoid duplicate code)